samples/modules/lvgl/demos/README.rst

```diff
@@ -0,0 +1,51 @@
+.. zephyr:code-sample:: lvgl-demos
+   :name: LVGL demos
+   :relevant-api: display_interface
+
+   Run LVGL built-in demos.
+
+Overview
+********
+
+A sample showcasing upstream LVGL demos.
+
+* Music
+      The music player demo shows what kind of modern, smartphone-like user interfaces can be created on LVGL.
+* Benchmark
+      The benchmark demo tests the performance in various cases. For example rectangle, border, shadow, text, image blending, image transformation, blending modes, etc.
+* Stress
+      A stress test for LVGL. It contains a lot of object creation, deletion, animations, styles usage, and so on. It can be used if there is any memory corruption during heavy usage or any memory leaks.
```

Comment:
nit: should these be wrapping on 80 columns or something? @kartben @gmarull not sure if it's a rule or guideline or just not a thing for doc files but usually everything in the repo is bounded it seems